### PR TITLE
Vmo 3121 tidy up builder embedding

### PR DESCRIPTION
**Pull request by bulatgab**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,14 +52,15 @@
...
-    "leader-line": "^1.0.5",
+    "leader-line": "^1.0.7",
     "lodash": "^4.17.19",
     "m-class-list": "^1.1.9",
     "moment": "^2.27.0",
     "node-sass": "^4",
     "plain-draggable": "^2.5.12",
     "pointer-event": "^1.0.0",
     "popper.js": "^1.16.1",
+    "skeleton-loader": "^2.0.0",
```
Comment (safydy-r) [inline package.json:63]: Can you guys give a summary why this is a solution?

**Review by safydy-r — APPROVED** [review]
Approving, thanks @bulatgab & @jcbashdown

**Review by jcbashdown — APPROVED** [review]
Looks good. Thanks Bulat!

File: package.json (hunk 1 above)
Comment (bulatgab) [reply package.json:63]: To my knowledge, the `leader-line` library is quite old and it doesn't support js modules, i.e. it doesn't have `export` in it. So we have to use a `skeleton-loader` to add it.

**Pull request merged**